src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -243,17 +243,29 @@ void AddValueDocumentationParts(ISymbol symbol, IDocumentationCommentFormattingS
 
             private void AddExceptions(ISymbol symbol)
             {
-                var exceptionTypes = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true).ExceptionTypes;
+                const int maxTextSize = 70;
+                var documentationComment = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true);
+                var exceptionTypes = documentationComment.ExceptionTypes;
+                var formatter = Services.GetLanguageServices(_semanticModel.Language).GetRequiredService<IDocumentationCommentFormattingService>();
                 if (exceptionTypes.Any())
                 {
-                    var parts = new List<SymbolDisplayPart>();
+                    var parts = new List<TaggedText>();
                     parts.AddLineBreak();
                     parts.AddText(WorkspacesResources.Exceptions_colon);
                     foreach (var exceptionString in exceptionTypes)
                     {
-                        parts.AddRange(LineBreak());
-                        parts.AddRange(Space(count: 2));
-                        parts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel));
+                        var checkedParts = new List<TaggedText>();
+                        checkedParts.AddRange(LineBreak().ToTaggedText());
+                        checkedParts.AddRange(Space(count: 2).ToTaggedText());
+                        checkedParts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel).ToTaggedText());
+                        var exceptionTexts = documentationComment.GetExceptionTexts(exceptionString);
+                        if (!string.IsNullOrEmpty(exceptionTexts[0]))
+                        {
+                            var exceptionStr = exceptionTexts[0].Length - 1 > maxTextSize ? $": {(exceptionTexts[0].Remove(maxTextSize - 3) + "...")}" : $": {exceptionTexts[0]}";
```

Comment:
After thinking about this, should this really be truncated instead of just limiting the # of exceptions displayed per page in the IDE?

Reply:
```suggestion
                            var exceptionStr = $": {exceptionTexts[0]}";
```